cli/lsp/analysis.rs

```diff
@@ -208,6 +227,23 @@ pub enum ResolvedDependency {
   Err(ResolvedDependencyErr),
 }
 
+impl ResolvedDependency {
+  pub fn as_hover_text(&self) -> String {
+    match self {
+      Self::Resolved(specifier) => match specifier.scheme() {
+        "data" => "_(a data url)_".to_string(),
```

Comment:
What are your thoughts on displaying a code block containing the decoded string of the data URL?

Reply:
Or too big to meaningful display. You can goto source on the module and on symbols exported from the URL, that is sufficient IMO.